src/bundle/Resources/public/scss/_breadcrumb.scss

```diff
@@ -0,0 +1,16 @@
+.breadcrumb {
+    background-color: $ez-ground-base-dark;
+}
+
+.breadcrumb-item {
+    cursor: not-allowed;
+    
+    + .breadcrumb-item::before {
+        color: $ez-color-base-medium;
+    }
+
+    &.active {
+        color: $ez-color-base-medium;
+        cursor: not-allowed;
```

Comment:
Why we need `cursor: not-allowed` on `.active`? The same rule is without `.active`.